raiden/constants.py

```diff
@@ -49,6 +58,14 @@ class EthClient(Enum):
     3: 3604000,  # 924 blocks before token network registry deployment
 })
 
+ID_TO_CONTRACT_ADDRESSES = {
+    3: {
+        CONTRACT_ENDPOINT_REGISTRY: to_canonical_address(ROPSTEN_DISCOVERY_ADDRESS),
+        CONTRACT_SECRET_REGISTRY: to_canonical_address(ROPSTEN_SECRET_REGISTRY_ADDRESS),
+        CONTRACT_TOKEN_NETWORK_REGISTRY: to_canonical_address(ROPSTEN_REGISTRY_ADDRESS),
```

Comment:
what about merging this map with this another map?

https://github.com/raiden-network/raiden/blob/2f1b0bccec1103e6eeba9bd0eb073f5abd3d5967/raiden/constants.py#L48-L50

Reply:
Done